### ISSUE TITLE
Feature request: Only include certain attributes

### ISSUE THREAD

**Issue by rejeep** [bjorne/jku#1]
Hello mr. Bjorn.

It's not in my nature to ask other people for favors (unless sexual), but I'm too lazy in this case.

I would like an option that works similar to transformers. The option takes one or more arguments and will for each JSON line, only include attributes that are specified as arguments to the option. For example:

``` json
{"one": "lorem", "two": "ipsum",     "three": "dolor"}
{"one": "amet",  "two": "consequat", "three": "tortor"}
{"one": "quis",  "two": "fermentum", "three": "magna"}
```

Running this:

``` bash
$ cat ABOVE-FILE | jku --some-option one three
```

Would produce this:

``` json
{"one": "lorem", "three": "dolor"}
{"one": "amet",  "three": "tortor"}
{"one": "quis",  "three": "magna"}
```

Could you please implement this for me sugar?

**Comment by bjorne** [comment]
You mean you don't like writing this?

```
$ cat ABOVE-FILE | jku -t 'Object.keys(_).filter(function(k) { return ["one", "three"].indexOf(k) != -1 }).reduce(function(m,k) { m[k] = _[k]; return m; }, {})'
```

I like the idea!

**Comment by rejeep** [comment]
Yes! If you are working with large Json files, the first step could be to only select the attributes you want and save that to a new file. That would be consecutive operations a lot faster.

It would be neat if it was a little more complex that what you do above so that you could do this for nested expressions, for example:

``` bash
$ cat SOME-FILE | jku --some-option one three names:names.people[0].name
```

There are most likely other complex situations that could be very useful.
